lib/tape/installer.rb

```diff
@@ -45,37 +48,16 @@ def create_ssh_keys_dir
     end
 
     def create_inventory_file
-      copy_example 'templates/base/hosts.example', "#{tapefiles_dir}/hosts"
+      copy_example "#{tape_dir}/templates/base/hosts.example", "#{tapefiles_dir}/hosts"
     end
 
-    def copy_static_app_examples
-      copy_example(
-        'templates/static_html/omnibox.example.yml',
-        "#{tapefiles_dir}/omnibox.yml"
-      )
-      copy_example(
-        'templates/static_html/deploy.example.yml',
-        "#{tapefiles_dir}/deploy.yml"
-      )
-      copy_example(
-        'templates/static_html/tape_vars.example.yml',
-        "#{tapefiles_dir}/tape_vars.yml"
-      )
-    end
-
-    def copy_basic_examples
-      copy_example(
-        'templates/base/omnibox.example.yml',
-        "#{tapefiles_dir}/omnibox.yml"
-      )
-      copy_example(
-        'templates/base/deploy.example.yml',
-        "#{tapefiles_dir}/deploy.yml"
-      )
-      copy_example(
-        'templates/base/tape_vars.example.yml',
-        "#{tapefiles_dir}/tape_vars.yml"
-      )
+    def copy_examples(type)
+      Dir["#{tape_dir}/templates/#{type}/*"].each do |file_name|
+        basename = File.basename file_name
+        basename.gsub! /\.example/, ""
```

Comment:
Ambiguous regexp literal. Parenthesize the method arguments if it's surely a regexp literal, or add a whitespace to the right of the `/` if it should be a division.